### ISSUE TITLE
Render code blocks with syntax highlighting

### ISSUE THREAD

**Issue by dns2utf8** [rust-lang/crates.io#1020]
Thank you for the cool feature.
I noticed that it currently does not render rust code with highlighting.

In one of [my crates](https://crates.io/crates/drop_guard) I have a code example with  a ´```rust´ block.
Can you extend it to highlight the code?

Best regards

**Comment by carols10cents** [comment]
We had to disable syntax highlighting temporarily for security reasons-- https://github.com/rust-lang/crates.io/issues/1008 we're waiting on the next Ammonia crate release to re-enable.

I'm going to close this as a duplicate of #1008 :)

**Comment by dns2utf8** [comment]
Works for me. Thank you.
